src/_guides/libraries/futures-error-handling.md

```diff
@@ -205,45 +205,56 @@ easiest to understand through examples.
 In the code below, `then()`'s Future completes with an error, so
 `whenComplete()`'s Future also completes with that error.
 
-{% prettify dart tag=pre+code %}
+<?code-excerpt "futures/lib/when_complete.dart (with-error)" replace="/withErrorMain/main/g; "?>
+```dart
 void main() {
   funcThatThrows()
-    .then((_) => print("won't reach here"))    // Future completes with an error.
-    .whenComplete(() => print('reaches here')) // Future completes with the same error.
-    .then((_) => print("won't reach here"))    // Future completes with the same error.
-    .catchError(handleError);                  // Error is handled here.
+      // Future completes with an error
+      .then((_) => print("Won't reach here"))
+      // Future completes with the same error
+      .whenComplete(() => print('Reaches here'))
+      // Future completes with the same error
+      .then((_) => print("Won't reach here"))
+      // Error is handled here
+      .catchError(handleError);
```

Comment:
We should have punctuations after the comments, no? Usually it'd be periods, but maybe in this case it should be colons (`:`).